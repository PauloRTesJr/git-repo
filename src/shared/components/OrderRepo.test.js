import React from "react";
import { render, cleanup } from "@testing-library/react";
import OrderRepo from "./OrderRepo";

afterEach(cleanup);

it("renders", () => {
    const { asFragment } = render(<OrderRepo />);
    expect(asFragment()).toMatchSnapshot();
});