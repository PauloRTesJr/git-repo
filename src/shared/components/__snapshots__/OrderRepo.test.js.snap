// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <select
    class="sc-bdVaJa elSbrI"
  >
    <option
      value="0"
    >
      Nome
    </option>
    <option
      value="1"
    >
      Seguidores
    </option>
    <option
      value="2"
    >
      Issues
    </option>
    <option
      value="3"
    >
      Stars
    </option>
  </select>
</DocumentFragment>
`;
